spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/README.md

```diff
@@ -4,6 +4,8 @@ This example demonstrates how to use BigDL to load pre-trained [Torch](http://to
 
 **ModelValidator** provides an integrated example to load models, and test over imagenet validation dataset on Spark.
 
+For most cnn models, mkldnn engine type will get better performance, so you can set bigdl.engineType property to mkldnn and have a try.
```

Comment:
For most CNN models, it's recommended to enable MKL-DNN acceleration capacity by specifying `bigdl.engineType `

Reply:
ok, changed